app/models/sample_transfer.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+# entity class for SampleTransfer
+class SampleTransfer
```

Comment:
Since the samples transfer service only takes 2 parameters we don't really need a special class for them. We can just pass the parameters directly to the `execute` method of the transfer service. So please remove this class and update accordingly

Reply:
I can modify the transfer service to only accept 2 params, but I created SampleTransfer to help with the form. SampleTransfer is an active model so I can call ` form_with(model: @sample_transfer)`. I liked being able to use a form like we do for our active records. Would you have me remove SampleTransfer entirely and not use an active model in the form at all?

Reply:
Yeah let's go with removing the `SampleTransfer` entirely. I don't think we should be using forms with models unless the model has a table attached to it. So yeah let's update the transfer service to accept 2 params

Reply:
Quick question. I've started removing `SampleTransfer`, I just don't know what I should be attaching the errors to be displayed on the page.

Reply:
I believe we can just attach them to the @new_project variable and then have the execute method return the @new_project to the controller

Reply:
I might not have a defined @new_project at the time the error occurs. I might need to reach out to you about this one... I'm sorry.

Reply:
As discussed, I am to append the errors to the current project. Please see 018e40ae38dddb1ed820bb0e72bc6ba699544a67.